docs/source/conf.py

```diff
@@ -43,6 +46,9 @@
 exclude_patterns = ['_build', 'Thumbs.db', '.DS_Store']
 source_suffix = {'.rst': 'restructuredtext', '.txt': 'restructuredtext', '.md': 'markdown'}
 
+# do not execute notebooks before including in the examples
+# assuming they were executed before
+nbsphinx_execute = 'never'
```

Comment:
why not auto?

Reply:
We expect the notebooks to be executed in the separate `nb-runner` routine and be executable in `mrpro[notebook]` environment. The docs runs in `mrpro[docs]` environment though and would not only probably crash in execution but also interfere with the result of the runners. This way we would not be able to check where the origin of the problem is. So we expect the runners to deliver a 100% correct executed notebooks for the docs.